libs/librepcb/project/boards/items/bi_footprintpad.cpp

```diff
@@ -118,6 +119,12 @@ bool BI_FootprintPad::isOnLayer(const QString& layerName) const noexcept {
   }
 }
 
+bool BI_FootprintPad::isTHT() const noexcept {
+  GraphicsLayer* padLayer =
+      mBoard.getLayerStack().getLayer(getLayerName());
+  return (padLayer && !padLayer->isCopperLayer());
```

Comment:
This looks a bit strange to me (although probably it works). I'd suggest to check the value of `mFootprintPad->getBoardSide()` instead. And actually I would not even add a method for this, since users of `BI_FootprintPad` could simply write `if (pad.getLibPad().getBoardSide() == FootprintPad::THT)` :wink:

An alternative would be to add `isTht()` to `library::FootprintPad` since then it would also be helpful for library related code. And for `BI_FootprintPad`, one could write `if (pad.getLibPad().isTht())` then.

In any way, for consistency reasons the method should be called `isTht()` instead of `isTHT()` :grin: